### PR TITLE
[OKTA-406025] chore: add memory usage command

### PR DESCRIPTION
**Pull request by giselleserate-okta**
* Add memory usage command
* Add basic tests for two cases

to be completely honest, I have no idea how the SAMPLES key is supposed to work. I verified that the behavior matched what happened when I ran it in redis-cli (and it doesn't error or panic or anything) but it never changed the number when I added SAMPLES to the query.

### CODE REVIEW

**Comment by giselleserate-okta** [comment]
Verified in my incoming redis script PR against alpha--that's still polishing, but this is ready for review

File: tests/integration/cluster.rs [hunk 1]
```diff
@@ -267,6 +268,28 @@ pub mod pubsub {
...
+  fn it_should_memory_usage_simple_db() {
+    let config = RedisConfig::default_centralized();
+    utils::setup_test_client(config, TIMER.clone(), |client| {
+      other_tests::memoryusage::should_memory_usage_simple(client)
+    })
+  }
+
+  #[test]
+  fn it_should_memory_usage_hset_db() {
+    let config = RedisConfig::default_centralized();
```
Comment (colingabrielson-okta) [inline tests/integration/cluster.rs:285]: Shouldn't this and the one above be:
```suggestion
    let config = RedisConfig::default_clustered();
```

Comment (giselleserate-okta) [reply tests/integration/cluster.rs:285]: ooops, thanks for catching that!

**Review by colingabrielson-okta — APPROVED** [review]
LGTM

**Pull request merged**